zipline/_protocol.pyx

```diff
@@ -21,6 +21,28 @@ from cpython cimport bool
 
 from zipline.assets import Asset
 
+
+class assert_keywords(object):
+    """
+    Asserts that the keywords passed into the wrapped function are included
+    in those passed into this decorator. If not, raise a TypeError with a
+    meaningful message, unlike the one cython returns by default.
```

Comment:
What does the Cython error that we are covering over look like?

Not a blocker for this patch, but is 'fixing' the Cython error something we should upstream?